flake.nix

```diff
@@ -12,6 +12,8 @@
       barretenbergOverlay = final: prev: {
         barretenberg = prev.callPackage ./barretenberg.nix { };
         barretenberg-wasm = prev.callPackage ./barretenberg-wasm.nix { };
+        barretenberg-wasm-debug = prev.callPackage ./barretenberg-wasm.nix { debug = true; };
```

Comment:
I'd rather us use `.override { debug = true }` in packages that want to consume barretenberg that we want to debug. Is there a reason you wanted to add another package?

Reply:
This way we can build it remotely and grab just debug artifact easily. Override patter would mean that you need to be changing configs in a workspace?